lib/less/parser/parser.js

```diff
@@ -953,7 +953,7 @@ var Parser = function Parser(context, imports, fileInfo) {
                             arg = entities.variable() || entities.property() || entities.literal() || entities.keyword() || this.call(true);
                         }
 
-                        if (!arg) {
+                        if (!arg || !hasSep) {
```

Comment:
Will this ever evaluate to true?  It looks like `hasSep` isn't touched between `hasSep = true` and `!hasSep`.  Unless there's some kind of non-linear code happening that would set it.

Reply:
It's set per loop. The first time it's true; that's intentional because it doesn't need a separator. Then it continues to be set by the presence of a separator, but depends on what type during the loop. So if ever it finishes without detecting a true separation of args, it should fail... I think. It seems to work? lol

Reply:
Good enough for me.